plugins/woocommerce/includes/customizer/class-wc-shop-customizer.php

```diff
@@ -249,6 +275,33 @@ public function add_scripts() {
 			} );
 		</script>
 		<?php
+
+		if ( $this->has_block_checkout() ) {
+			$message = sprintf(
+				/* translators: %s: Link to the editor page with the Checkout block. */
+				__( 'Checkout can be customized <a href="%s">in the Editor</a> with your active theme.', 'woocommerce' ),
+				admin_url( 'post.php?post=' . get_option( 'woocommerce_checkout_page_id' ) . '&action=edit' )
+			);
+			?>
+			<script type="text/javascript">
+				jQuery( document ).ready( function( $ ) {
+					const message = <?php echo wp_json_encode( $message ); ?>;
+					console.log(message);
+					$( "#sub-accordion-panel-woocommerce" ).append( "<li class='notice notice-info'>" + message + "</li>" );
+				} );
+			</script>
+			<?php
+		}
+
+	}
+
+	/**
+	 * Enqueue scripts for the customizer.
+	 */
+	public function enqueue_scripts() {
+		$handle = 'custom-notice';
+		wp_register_script( $handle, false, array( 'customize-controls' ), WC_VERSION, false );
+		wp_enqueue_script( $handle );
```

Comment:
This section is needed to load the required dependencies. Without loading `customize-controls`, the notice will not appear.